resume.qmd

```diff
@@ -0,0 +1,67 @@
+---
+title: "resume"
+editor: visual
+---
+
+## Joshua Yamamoto
+
+**Portland, OR • github: joshyam-k • yamamojo\@reed.edu**
```

Comment:
Since this is a `.qmd`, could link your GitHub explicitly. Also, maybe include an email that will persist after you graduate?